index.js

```diff
@@ -8,6 +8,7 @@ class ServerlessPlugin {
     ];
 
     this.hooks = {
+      'before:offline:start': this.startHandler.bind(this),
```

Comment:
@XuluWarrior can you verify that adding this and leaving in `before:offline:start:init` doesn't cause `startHandler` to execute twice?

Reply:
I can confirm that
- if you run `sls offline` then only the `before:offline:start` hook fires.
- if you run `sls offline start` then only the `before:offline:start:init` hook fires.

So `startHandler` is only called once in both cases.

I haven't tested any other ways of starting up, if such ways exist.

Reply:
Sounds good!